packages/serverpod/lib/src/relic/web_server.dart

```diff
@@ -126,17 +126,26 @@ class WebServer {
       }
     }
 
-    // TODO: Fix body
-    var session = MethodCallSession(
-      server: serverpod.server,
-      uri: uri,
-      path: 'webserver',
-      body: '',
-      authenticationKey: authenticationKey,
-      httpRequest: request,
-    );
-
-//    print('Getting path: ${uri.path}');
+    MethodCallSession session;
+    try {
+      // TODO: Fix body
+      session = MethodCallSession(
+        server: serverpod.server,
+        uri: uri,
+        path: 'webserver',
+        body: '',
+        authenticationKey: authenticationKey,
+        httpRequest: request,
+      );
+    } catch (e) {
+      // Triggered if the URI query parameters are malformed
+      if (serverpod.runtimeSettings.logMalformedCalls) {
+        logDebug('Malformed call: $e');
```

Comment:
Suggestion: don't use logDebug since it just logs to standard out. Please use `stderr.writeln` since this is an error.